charts/lfx-v2-indexer-service/templates/deployment.yaml

```
@@ -7,6 +7,8 @@ kind: Deployment
 metadata:
   name: lfx-v2-indexer-service
   namespace: lfx
+  labels:
+    app.kubernetes.io/name: {{ .Chart.Name }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)